src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  &__icon {
+    width: 40px;
+    height: 40px;
+    img {
+      display: block;
+    }
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    padding: 0;
+    margin: 0;
+  }
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  &__link.is-active::after {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    content: '';
+    background-color: #00acdc;
+    width: 100%;
+    height: 4px;
+    border-radius: 8px;
+  }
+  &__item:last-child {
+    margin-right: 0;
+  }
+  &__item a {
+    text-decoration: none;
+    color: #060b35;
+    text-transform: uppercase;
+    transition: all 0.3s;
+    font-size: 12px;
+    height: 60px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
The `&__item a` selector is a bit unnecessary, as you already have a `&__link` selector. It would be more appropriate to move these styles to the `&__link` selector to keep your code DRY (Don't Repeat Yourself). Also, it will make your code more readable and easier to maintain.